ts/components/styles.ts

```diff
@@ -63,3 +68,152 @@ export const ProfileStyles = StyleSheet.create({
     textAlign: "right"
   }
 });
+
+export const PortfolioStyles = StyleSheet.create({
```

Comment:
these should all come from the theme, see for instance how [this component](https://github.com/teamdigitale/italia-app/blob/master/ts/components/Pinpad.tsx) takes colors and font sizes from `variables.ts`

Reply:
cc @valentinacala consider this when working on the styles :)